ext/flash/01_http.js

```diff
@@ -512,7 +531,7 @@
                 }
                 ws[_serverHandleIdleTimeout]();
               }
-            })().catch(onError);
+            })().catch(console.error);
```

Comment:
i think this was mentioned before. need to verify exactly what kind of errors we expect to get here and how to handle them and what to do if we get something unexpected. in general, this function seems pretty *huge* and am wondering does v8 ever fully optimise it? i can do some testing to verify, or do we have some special optimisation trick for these internal modules?

Reply:
oops that change wasn't intentional, reverted. yeah, maybe we should refactor out the whole request handling logic in a function